kubeshop/monokle
# Wito/feat/improve validation error hovers

This PR closes #1627. It finalises rules and improves the experience within the editor.

## Changes

- Update rules
- Improve hover
- Improve error position determination

## How to test it

- Open a project with policy errors.
- Open the editor and see errors being highlighted

## Checklist
